.kitchen.vagrant.yml

```diff
@@ -1,44 +0,0 @@
----
-driver:
```

Comment:
I would prefer to have:
- `.kitchen.yml` - Docker-based
- `.kitchen.vagrant.yml` - vagrant based
- `.kitchen.digitalocean.yml` - in digital ocean
This would allow all users to run the tests locally as well.